modules/admin_manual/pages/enterprise/external_storage/ldap_home_connector_configuration.adoc

```diff
@@ -3,59 +3,122 @@
+
+For enhanced security, create a file where the credentials are stored accessing the cifs share like:
+
+`/etc/credentials`
+
+with the username and password on separate lines, replacing the values according your setup:
+
+[source,text]
 ----
 username=winhomeuser
 password=winhomepassword
 ----
 
-Then add a line like this to /etc/fstab, substituting your own server address and filenames:
+Create an entry in `/etc/fstab` for the remote Windows root home directory mount and use the credentials file created above, substitute and adopt your parameters and filenames:
 
+[source,text]
 ----
-//192.168.1.58/share /mnt/share cifs credentials=/etc/credentials,uid=33,gid=33
+//192.168.1.58/home /mnt/share/users cifs credentials=/etc/credentials,uid=33,gid=33
 ----
 
-== Configure the LDAP Home Connector
+=== Configure the LDAP Server
 
-Enable the LDAP Home Connector app. Then go to the LDAP Home Connector
-form on your ownCloud admin page. In the *Display folder as:* field
-enter the name as you want it to appear on your users’ File pages.
+In Active Directory, open the user profile. Scroll to the *Extensions* section and open the *Attribute Editor* tab
 
-Then in the *Attribute name:* field enter the LDAP attribute name that
-will contain the home directory. Use any LDAP attribute that is not
-already in use, then save your changes.
+image:enterprise/external_storage/ldap-home-connector/ldap-home-connector-2.png[Active Directory Attribute editor.]
 
-image:enterprise/external_storage/ldap-home-connector/ldap-home-connector-1.png[LDAP Home Connector configuration.]
+Use any LDAP attribute that is not already in use (UserSharedFolder in this instance), and click *Edit*. Enter the users home directory.
 
-== Configure the LDAP Server
+image:enterprise/external_storage/ldap-home-connector/ldap-home-connector-3.png[Editing the LDAP attribute.]
 
-In Active Directory, open the user profile. Scroll to the *Extensions*
-section and open the *Attribute Editor* tab
+Save your changes.
 
-image:enterprise/external_storage/ldap-home-connector/ldap-home-connector-2.png[Active Directory Attribute editor.]
+=== Configure the LDAP Home Connector
 
-Scroll to the attribute being used (UserSharedFolder in this instance),
-and click *Edit*. Enter the users home directory.
+* Enable the LDAP Home Connector app.
+* Go to the LDAP Home Connector form on your ownCloud admin page. In the *Display folder as:* field enter the name as you want it to appear on your users’ File pages.
+* In the *Attribute name:* field enter the LDAP attribute name from above that contains the home directory and press btn:[Save].
 
-image:enterprise/external_storage/ldap-home-connector/ldap-home-connector-3.png[Editing the LDAP attribute.]
+image:enterprise/external_storage/ldap-home-connector/ldap-home-connector-1.png[LDAP Home Connector configuration.]
 
-Save your changes, and you are finished.
+The Windows users home directory is now available to the user when he logs on in ownCloud.
```

Comment:
```suggestion
The Windows users home directory is now available to the user when they log on in ownCloud.
```

I think we use "they" rather than "he" or "she" these days?